src/Mutation/MediaItemUpdate.php

```diff
@@ -145,8 +145,13 @@ public static function mutate_and_get_payload() {
 			 */
 			$post_id = wp_update_post( $clean_args, true );
 
-			if ( 0 === $post_id || is_wp_error( $post_id ) ) {
-				throw new UserError( __( 'The media item failed to update', 'wp-graphql' ) );
+			if ( is_wp_error( $post_id ) ) {
```

Comment:
`0 === $post_id` should be added back: https://developer.wordpress.org/reference/functions/wp_update_post/